runtime/j9vm/jvm.c

```diff
@@ -1567,6 +1571,22 @@ waitForJITServerTermination(JITServer *jitServer)
 	}
 	return rc;
 }
+
+/**
+ * Stops the JITServer.
+ *
+ * @param jitServer pointer to the JITServer interface
+ *
+ * @returns void
+ */
+static void
+stopJITServer(JITServer *jitServer)
+{
+	JavaVM *vm = jitServer->jvm;
+	(*vm)->DestroyJavaVM(vm);
```

Comment:
Does this actually stop the server?  DestroyJavaVM typically waits until there are no-non daemon threads - are you intentionally depending on that behaviour?  Would be good to add a comment here if that's the intention

Reply:
From JITServer point of view we only want to terminate the Listener thread and we get this chance when DestroyJavaVM is invoked (`protectedDestroyJavaVM` -> `runExitStages` -> `jitExclusiveVMShutdownPending` -> `listener->stop`).

Reply:
Then I'm very unclear on the way to use the APIs here.  When does `stopJITServer` get called vs `waitForJITServerTermination`?

Reply:
My reply [here](https://github.com/eclipse/openj9/pull/8844#discussion_r398071090) probably addresses your question to some extent.